Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN go run main.go -docFolder=../docs
 FROM alpine:3.11
 
 COPY --from=builder /docs /docs
+COPY docs/tool-description.md /docs/
```

Comment:
From what I see in other tools (also written in scala) the first line just doesn't have "from builder" thingy 
https://github.com/codacy/codacy-faux-pas/blob/master/Dockerfile
https://github.com/codacy/codacy-bandit/blob/master/Dockerfile

Reply:
anyway I believe @lolgab knows better 😅  I am just shooting in the dark

Reply:
For some reason, this tool uses the image `golang:1.14.4-alpine3.11` to build the documentation for the patterns, and was copying just the results of the command `go run main.go -docFolder=../docs` that don't include the description file.

My line ensures that the `tool-description.md` file is also copied into the final generated Docker image.

Reply:
As @prcr explained already, the `--from=builder` flag make the COPY take the folder from the `builder` stage instead of the repository directory.
This is useful when you run the documentation generator inside the `builder` stage (like here in this tool)

Reply:
@lolgab but then is paulo suggestion the best to make this work?